ocaml/xapi/xapi_globs.ml

```diff
@@ -930,7 +930,7 @@ let repository_gpgkey_name = ref ""
 
 let repository_gpgcheck = ref true
 
-let ignore_vtpm_unimplemented = ref false
+let ignore_vtpm_unimplemented = ref true
```

Comment:
can this be turned to false and removed in the snapshot case?

Reply:
So keep it and just remove the checks? Will look for it